content/en/docs/reference/glossary/namespace-sameness.md

```diff
@@ -5,5 +5,5 @@ test: n/a
 
 Within a multicluster mesh, [namespace sameness](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md)
 applies and all namespaces with a given name are considered to be the same namespace. If multiple clusters contain a
-`Service` with the same namespaced name, they will be recognized as a single combined service. By default, traffic is
```

Comment:
Do you know what other pages you looked at and didn't find this information?

This is the glossary definition for namespace sameness.  What about "same namespaced name and port name"?

Reply:
I mean `Service.spec.ports[_].name`, as pined a link https://github.com/istio/istio/blob/93e4ec424b52da807228bd7784c59d646ec73733/pilot/pkg/xds/endpoint_builder.go#L240-L242

if the multicluster `Service` have same port, but each with a different name, they are no longer `same`, only the service in primary cluster affect.

```yaml
apiVersion: v1
kind: Service
metadata:
  name: nginx-service
spec:
  selector:
    app.kubernetes.io/name: proxy
  ports:
  - name: name-of-service-port       # <-- I mean this, service.spec.ports[_].name
    protocol: TCP
    port: 80
    targetPort: http-web-svc
```

Reply:
I understand the situation, but I'm just trying to clarify: "namespace sameness" is a thing that exists in Kubernetes, and what Istio is doing is saying "service and namespace sameness" is required.  Let me propose an edit for this.